landingai/data_management/client.py

```diff
@@ -167,15 +185,16 @@ async def _api_async(
             async with method(
                 endpoint,
                 headers=headers,
-                json=resp_with_content,
+                json=resp_with_content if not is_form_data else None,
                 params=params,
+                data=form if is_form_data else None,
                 ssl=True,
             ) as resp:
                 _LOGGER.debug("Request URL: ", resp.request_info.url)
                 _LOGGER.debug("Response Code: ", resp.status)
                 _LOGGER.debug("Response Reason: ", resp.reason)
 
-                if resp.status != 200:
+                if not re.match(r"2[0-9]{2}$", str(resp.status)):
```

Comment:
It is cheaper (and probably more readable) to do something like `if resp.status // 100 != 2`